wordle-web/src/views/AboutView.vue

```diff
@@ -1,17 +1,38 @@
 <template>
-  <v-sheet color="blue" height="170px" class="pt-10 my-5">
-    <v-card color="indigo-darken-3" class="ma-5" elevation="5" rounded="xl">
-      <v-card-title>Hello World</v-card-title>
+  <v-sheet
+    location="center"
+    color="#9E9E9E"
+    height="auto"
+    max-width="800px"
+    class="pt-10 my-5"
+    rounded="lg"
+  >
+    <v-card color="cyan-darken-3" class="ma-5" elevation="5" rounded="xl">
+      <v-card-title>About Us</v-card-title>
+      <v-card-text>
+        Hello! Our names are Nolan Posey and Hunter Thompson. This web app is a project we are
+        working on for our CSCD 379 .NET Web Applications class at Eastern Washington University.
+        The goal of this project is to emulate the game Wordle and to apply our own custom flair to
+        the game in order to really make it our own. The name of our game is called "The Good Word"
+        and we hope you enjoy playing it as much as we enjoying making it!
+      </v-card-text>
+    </v-card>
+
+    <v-card color="cyan-darken-3" class="ma-5" elevation="5" rounded="xl">
+      <v-card-title>Our Teacher Meg</v-card-title>
+      <v-card-text>
+        Meg has been teaching us CSharp and Vuetify since Winter quarter of the 2022-2023 school
+        year. Her ability and passion to teach has helped us hone the necessary skills we as
+        programmers need in order to be successful in our careers. Meg is over-worked and
+        under-paid, but since we have no money... and no influence over her bosses Mark and Grant...
+        we have decided to gift her with this cute image Nolan's cat Nala.
```

Comment:
My work anniversary is coming up, so I'll be sure to forward this comment about me needing a raise to Mark and Grant. :) 

Here is my cat and a half in return.

![image](https://user-images.githubusercontent.com/43832670/236657200-44e62186-55a1-4d7d-9980-6dfc905366e6.png)